src/_ert/forward_model_runner/reporting/event.py

```diff
@@ -1,9 +1,9 @@
 from __future__ import annotations
 
+import asyncio
 import logging
 import queue
-import threading
-from datetime import datetime, timedelta
+import time
```

Comment:
Can we remove this import and use `asyncio.time()` instead?

Reply:
Yes, it can be replaced. Will do :+1: